Master

Amantel opened the pull request:
OK... my first pull request. I hope this is fine

Amantel committed:
Got totalRoot for one item group

Amantel committed:
Marked root nodes for first result example

Amantel committed:
WishList problems  - loosing wishlist in result 0

Amantel committed:
Only first parent as a card? Error(

Amantel committed:
Fixed Card finder v1

Amantel committed:
New CardFinder

Amantel committed:
Find all cards exept for mans/childern

Test need.
TODO: add IDs to target cards

Amantel committed:
+IDs for cards for tests

Amantel committed:
+ visual tests

Amantel committed:
Updated DOM for tests

Amantel committed:
Some additions on images

Amantel committed:
Added Fuzzy flag to regexp

Amantel committed:
Added extention AND new tests

SergeyTru commented on uri_template.js:
```diff
@@ -302,5 +308,15 @@ var templateForUris = function(url1, url2) {
+    ) || (
+      (path2.match(/\d\..*$/)!==null) &&  
+      (path1.match(/\d\..*$/)===null)
+      )  
```
во-первых, что это за магические регекспы?
во-вторых (path1.match(/\d\..*$/)===null) !== (path2.match(/\d\..*$/)===null)

Amantel committed:
Merge remote-tracking branch 'refs/remotes/SergeyTru/master'

Amantel committed:
Put links from different groups to card

A lot of fixes
Bash hack line 391 in futurator_on_page
New function (as in header) generateCardNodesList

Amantel committed:
Temp array for nodes

Чтобы не менять настоящие ноды новыми свойствами

Amantel committed:
Fixed problem with перезаписывающиеся Nodes

Amantel committed:
Fixed less then 2 nodes bug with pseudoCards
